integration-tests/features/stack_analyses.feature

```diff
@@ -134,3 +134,17 @@ Feature: Analyse API functionality check
      And I should find that valid outliers are reported
      And I should get license_analysis field in stack report
      And I should find the security node for all dependencies
+
+  Scenario: Check that the stack-analyses returns a valid response for dynamic manifest files
+    Given System is running
+    When I acquire the authorization token
+    Then I should get the proper authorization token
+    When I wait 10 seconds
+    When I tried to fetch dynamic manifests from s3
+    Then I should be able to validate them
+    When I send NPM package manifest valid_manifests/npmlist.json to stack analysis version 3 with authorization token
+    Then I should get 200 status code
+    When I send Python package manifest valid_manifests/pylist.json to stack analysis version 3 with authorization token
+    Then I should get 200 status code
+    When I send new Maven package manifest valid_manifests/dependencies.txt to stack analysis version 3 with authorization token
+    Then I should get 200 status code
```

Comment:
might be good to check the content of stack analysis

Reply:
File contents are already validated in stack analyses API,
i feel its kind of redundant here, as
Tests will Fail in either case:
a) We write a function to check file contents and contents are not valid.
b) We send the file as it is for Stack Analyses and hence server will throw an Error if file is not valid.
Please suggest.

Reply:
I meant that we should check the result of stack analysis, ie. the schema, whether all packages are mentioned etc. etc.

Reply:
@tisnik Added more scnerios. Please re-review